automation/test.sh

```diff
@@ -27,6 +27,21 @@ readonly TEMPLATES_SERVER="gs://kubevirt-vm-images"
 readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt}"
 
 
+# Skip if it's docs changes only
+env
+CI_GIT_BASE_REF=${PULL_BASE_REF:-main}
+CI_GIT_ALL_CHANGES=$(git diff --name-only $CI_GIT_BASE_REF $PULL_PULL_SHA)
+CI_GIT_SKIP_CHANGES=$(git diff --name-only $CI_GIT_BASE_REF $PULL_PULL_SHA | grep -E "^(docs/|.*/OWNERS|.*\.md)" || :)
+if [[ "$CI_GIT_ALL_CHANGES" == "$CI_GIT_SKIP_CHANGES" ]]; then
+    echo "Aborting tests (in a good way) as there were only none-code changes detected:"
+    echo -e "$CI_GIT_ALL_CHANGES"
+    echo "(Flakes will be found with prediodics)"
+    exit 0
+else
+    echo "Detected changes:"
+    echo -e "$CI_GIT_ALL_CHANGES"
+fi
```

Comment:
dont think we need this on the long run, wdyt ?

Reply:
Agreed